### PR TITLE
Points icons

### PR DESCRIPTION
**Pull request by nucleogenesis**
## Description

Adds the leaf icon and re-pre-compiles the icons and updates the rst definitions. This was required to fully remove the dependency on SVG Icon Inline Loader which included the huge Material Design icon package in its dependencies. KDS has it in the devDependencies and we specifically only bundle used icons.

The primary motivator for this was to ensure people don't run into timeout issues while running `yarn install`.

#### Issue addressed

Fixes Kolibri https://github.com/learningequality/kolibri/issues/7890

## Steps to test

Check the icons in the testing server blow.  Note that I've included them in the colored icons set even though we don't actually give it the color that is used for the fill.

## Reviewer guidance

- [ ] Is the code clean and well-commented?
- [ ] Are there tests for this change?
- [ ] Are all UI components LTR and RTL compliant (if applicable)?
- [ ] _Add other things to check for here_

### CODE REVIEW

File: docs/rstIconReplacements.txt [hunk 1]
```diff
@@ -78,6 +78,8 @@
...
+.. |pointsInctive| replace:: :raw-html:`<span class="design-system-icon design-system-icon-color-text"><svg role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="363.6 113 93.6 93.5"><path fill="#AAA" d="M455 115c-6.5-6.5-60.6 3.9-76.1 19.4-12.6 12.6-13.5 33.7-2.4 49.3l-12.9 9.6 13.2 13.2 9.9-12.8c16.1 11.7 36.5 10.1 49-2.5 15.9-16.1 26.2-69.3 19.3-76.2z"/></svg></span>`
```
Comment (radinamatic) [inline docs/rstIconReplacements.txt:82]: Typo here, should be `|pointsInactive|`

Comment (nucleogenesis) [reply docs/rstIconReplacements.txt:82]: That one is on line 82 no?

Comment (radinamatic) [reply docs/rstIconReplacements.txt:82]: Yep, line 82!

**Comment by radinamatic** [comment]
All icons display correctly, tested on FF, Chrome & Edge on Windows, and FF & Chrome on Ubuntu.

A part from the `|pointsInctive|` typo, the only thing I don't understand why is that the copy/duplicate icon still full in the netlify.app, but when I use the replacements in the docs, the icon is just outlined 😕 

Icons - Kolibri Design System — Mozilla Firefox_047


LEDev21 (start)  Running  - Oracle VM VirtualBox_046

File: docs/rstIconReplacements.txt (hunk 1 above)
Comment (nucleogenesis) [reply docs/rstIconReplacements.txt]: I honestly didn't see the misspelling until the third time I looked at it... :sweat_smile:

Comment (radinamatic) [reply docs/rstIconReplacements.txt]: 🧠 filling in the 🕳️ even when we might not want it to... 😛

**Comment by nucleogenesis** [comment]
@radinamatic is there another issue for the copy icons still open? I want to get this merged so https://github.com/learningequality/kolibri/pull/7904 can be merged, but don't want to miss addressing that issue more thoroughly.

**Comment by radinamatic** [comment]
@nucleogenesis I was hoping this PR would also address the icons listed in #141

**Comment by nucleogenesis** [comment]
@radinamatic will merge this and address the copy icons in #141 as it is assigned to me this sprint

**Comment by radinamatic** [comment]
Gotcha, will wait for it then!

Do you have an idea why I get different aspect of the icon in the docs, compared to what is in the KDS app? 😕 (see above)

**Comment by nucleogenesis** [comment]
@radinamatic I do not - but I'll be investigating that in #141 for sure

**Pull request merged**